tests/devices/i04/test_murko_results.py

```diff
@@ -21,6 +21,10 @@
 @patch("dodal.devices.i04.murko_results.StrictRedis")
 async def murko_results(mock_strict_redis: MagicMock) -> MurkoResultsDevice:
     murko_results = MurkoResultsDevice(name="murko_results")
+    murko_results.stop_angle.get_value = AsyncMock()
+    murko_results.stop_angle.get_value.return_value = 350
+    murko_results.invert_stop_angle.get_value = AsyncMock()
+    murko_results.invert_stop_angle.get_value.return_value = False
```

Comment:
Could: If you make this as a mock device then all of this is done for you:
```suggestion
    with init_devices(mock=True):
        murko_results = MurkoResultsDevice(name="murko_results")
```
Then in your tests you can do `set_mock_value(murko_results.stop_angle, 100)`. Leaving this as a could because I know these signals are going to be removed shortly anyway but good to know for future.